File: docs/archive/print_sheet_18_cols.py
import json

with open('live_metadata_utf8.json', 'r', encoding='utf-8') as f:
    text = f.read()

parts = text.split('--- ')
columns = []
for part in parts:
    if part.startswith('COLUMNS ---'):
        content = part.split('---')[0].split('\n', 1)[1].strip()
        columns = json.loads(content)

sheet_18_cols = [c for c in columns if c['sheet_id'] == 18]
print("Sheet 18 columns in Supabase:")
for c in sheet_18_cols:
    print(c)

File: docs/archive/print_sheet_18_cols.js
const fs = require('fs');

const metadataRaw = fs.readFileSync('live_metadata_utf8.json', 'utf8');
const parts = metadataRaw.split('--- ');
let columns = [];

for (const part of parts) {
  if (part.startsWith('COLUMNS ---')) {
    const jsonStr = part.substring(part.indexOf('\n')).split('---')[0].trim();
    columns = JSON.parse(jsonStr);
  }
}

const sheet_18_cols = columns.filter(c => c.sheet_id === 18);
console.log("Sheet 18 columns in Supabase:", sheet_18_cols);
